tests/test_adobe_vendor_id.py

```diff
@@ -749,17 +753,34 @@ def test_decode_round_trip_with_intermediate_mischief(self):
         eq_(("http://my-library.org/", "Patron identifier"), decoded)
 
     def test_encode(self):
-        """Test that _encode gives a known value with known input."""
+        # Test that _encode gives a known value with known input.
+
+        # Three pieces of data go into our JWT.
         patron_identifier = "Patron identifier"
         now = datetime.datetime(2016, 1, 1, 12, 0, 0)
         expires = datetime.datetime(2018, 1, 1, 12, 0, 0)
+
+        raw_jwt = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJodHRwOi8vbXktbGlicmFyeS5vcmcvIiwic3ViIjoiUGF0cm9uIGlkZW50aWZpZXIiLCJpYXQiOjE0NTE2NDk2MDAuMCwiZXhwIjoxNTE0ODA4MDAwLjB9.wKAnFfJVfJP55CIyD7PntFZrtWVTwDcXHjL-quTndzc'
```

Comment:
The JWT changes here because using Python 3 (or using OrderedDict) changes the order of the key-value pairs in the dict we're serializing, which changes the payload.

This took a long time to debug so I included the code we used to debug as part of the test.